packages/hurl/src/report/mod.rs

```diff
@@ -225,13 +225,17 @@ fn create_html_result(result: HurlResult) -> html::Element {
     } else {
         "failure".to_string()
     };
+    let relative_input_file = match result.filename.strip_prefix('/') {
+        None => result.filename.as_str(),
+        Some(value) => value,
+    };
     html::Element::NodeElement {
         name: "tr".to_string(),
         attributes: vec![
             html::Attribute::Class(status.clone()),
             html::Attribute::Data("duration".to_string(), result.time_in_ms.to_string()),
             html::Attribute::Data("status".to_string(), status.clone()),
-            html::Attribute::Data("filename".to_string(), result.filename.clone()),
```

Comment:
I think line 242 below needs a similar update to correctly link the href.

Reply:
you're right!

Reply:
I keep it in draft until I add more tests.
Thanks